spec/lib/coach/handler_spec.rb

```diff
@@ -5,27 +5,30 @@
 require "coach/errors"
 
 describe Coach::Handler do
+  subject(:handler) { described_class.new(terminal_middleware, handler: true) }
+
   let(:middleware_a) { build_middleware("A") }
   let(:middleware_b) { build_middleware("B") }
   let(:middleware_c) { build_middleware("C") }
   let(:middleware_d) { build_middleware("D") }
 
   let(:terminal_middleware) { build_middleware("Terminal") }
-  subject(:handler) { Coach::Handler.new(terminal_middleware, handler: true) }
 
   before { Coach::Notifications.unsubscribe! }
 
   describe "#call" do
-    let(:a_spy) { spy("middleware a") }
-    let(:b_spy) { spy("middleware b") }
```

Comment:
Wow. This was some of the first rspec I'd ever written. I forgot spy's even exist!